packages/@aws-cdk/aws-cloudwatch/test/test.dashboard.ts

```diff
@@ -149,6 +149,24 @@ export = {
     // THEN
     expect(stack).to(haveResource('AWS::CloudWatch::Dashboard', {}));
 
+    test.done();
+  },
+
+  'throws if DashboardName is not valid'(test: Test) {
+    // GIVEN
+    const app = new App();
+    const stack = new Stack(app, 'MyStack');
+
+    // WHEN
+    const toThrow = () => {
+      new Dashboard(stack, 'MyDashboard', {
+        dashboardName: 'My Invalid Dashboard Name',
+      });
+    };
+
+    // THEN
+    test.throws(() => toThrow());
```

Comment:
Verify error message (partially is good enough):

```ts
test.throws(() => toThrow(), /It can only contain alphanumerics/);
```

Reply:
Done, thanks